### PR TITLE
Import Firefox 66 beta3 schema.

### LINKED ISSUE #2418: Import Firefox 66 beta schemas

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Fixes #2418

### CODE REVIEW

File: src/schema/imported/manifest.json [hunk 1]
```diff
@@ -104,7 +104,7 @@
                   "enum": [
                     "spanning"
                   ],
-                  "onError": "warn"
+                  "default": "spanning"
```
Comment (rpl) [inline src/schema/imported/manifest.json]: Just as a side note, this change means that extensions with "not_allowed" or "split" as the requested incognito mode will start to get errors when submitted on AMO.

The error message we currently report for this validation error is unfortunately not super useful, as it doesn't currently say which are the allowed values (currently just "spanning"):
```
Code           Message                           Description                           File   Line   Column
JSON_INVALID   "/incognito" should be equal to   Your JSON file could not be parsed.
               one of the allowed values
```

Shane checked with Mike how many of such extension we have, and they should be very few (I recall there were less then 10).

Anyway, Firefox will also raise an error an refuse to install the extension, and so the linter and AMO behavior is going to be consistent with Firefox.

**Comment by wagnerand** [comment]
That test is flaky, I have seen it failing before. Restarting should fix it.

**Comment by rpl** [comment]
> That test is flaky, I have seen it failing before. Restarting should fix it.

@wagnerand good, that's what I suspected, thanks for confirming it! 
do you know if it is flaky only on appveyor or also on travis?

**Comment by wagnerand** [comment]
I don't remember exactly, sorry.

**Pull request merged**